src/base/ModuleImport.zig

```diff
@@ -55,36 +54,35 @@ pub const Store = struct {
         ident_store: *Ident.Store,
         gpa: std.mem.Allocator,
     ) Store {
-        var modules = List.init(gpa);
-        _ = modules.append(Self{
+        var modules = List{};
+        _ = modules.append(gpa, Self{
             .name = &.{},
             .package_shorthand = null,
-            .exposed_idents = collections.SafeList(Ident.Idx).init(gpa),
+            .exposed_idents = .{},
             .resolved = null,
         });
 
         for (builtin_names) |builtin| {
-            _ = modules.append(Self{
+            _ = modules.append(gpa, Self{
                 .name = builtin,
                 .package_shorthand = null,
-                .exposed_idents = collections.SafeList(Ident.Idx).init(gpa),
+                .exposed_idents = .{},
                 .resolved = null,
             });
         }
 
         return Store{
             .imports = modules,
             .ident_store = ident_store,
-            .gpa = gpa,
         };
     }
 
-    pub fn deinit(store: *Store) void {
+    pub fn deinit(store: *Store, gpa: std.mem.Allocator) void {
         for (store.imports.items.items) |*import| {
-            import.exposed_idents.deinit();
+            import.exposed_idents.deinit(gpa);
```

Comment:
Not for this PR, but every time I see this it sticks out. I think it is our only 1 potato 2 potato deinit where we are doing things one at a time instead of in batch.

Reply:
Agreed, would love to find a better pattern. Luckily this is correct behavior, so not terrible for now.